packages/quantic/force-app/main/default/lwc/exampleSearch/resultTemplates/childTemplate.html

```diff
@@ -0,0 +1,29 @@
+<template>
+  <c-quantic-result-template is-any-preview-open={isAnyPreviewOpen} hide-bottom-border result-preview-should-not-be-accessible={resultPreviewShouldNotBeAccessible}>
```

Comment:
Remove the `hide-bottom-border` attribute.

```suggestion
  <c-quantic-result-template is-any-preview-open={isAnyPreviewOpen} result-preview-should-not-be-accessible={resultPreviewShouldNotBeAccessible}>
```